docs/upgrading-gatekeeper.md

```diff
@@ -20,6 +20,8 @@ and skipping `v3.4.Z`.
 GATEKEEPER_PREV_VERSION=$(awk '/^GATEKEEPER_VERSION \?= .*/ {print $3}' Makefile)
 GATEKEEPER_VERSION=<DESIRED_VERSION>
 sed -i "s/GATEKEEPER_VERSION ?= .*/GATEKEEPER_VERSION ?= ${GATEKEEPER_VERSION}/" Makefile
+sed -i "s/VERSION ?= .*/VERSION ?= ${GATEKEEPER_VERSION}/" Makefile
```

Comment:
Is this doc still accurate after @dhaiducek's changes in #29 to use the version files?